week3/src/components/Card.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+// import theme from "./theme"
+
+
+export default function Card({ data, handleClick }) {
+    const { id, name, img, status, idx } = data
+
+    const handleCard = (idx) => {
+        if (handleClick) handleClick(idx)
```

Comment:
제 소소한 팁인데 이런 단일 조건문에서는 다음과 같이 JS의 단락평가를 이용해서 더 간소화할 수 있답니다!!
```suggestion
        handleClick && handleClick(idx)
```
handleClick이 true일 때는 handleClick && handleClick(idx)의 결과가 handleClick(idx)에 의해 결정되기 때문에 handleClick(idx)까지 동작해보면서 평가를 하기 때문에 handleClick(idx)가 동작하는데, handleClick이 false일 때는 handleClick(idx)가 무엇이든지 상관없이 handleClick && handleClick(idx) 자체가 false이므로 handleClick(idx)를 평가하지 않아버려서 실행되지 않는 원리입니다!!